fe/fe-core/src/main/java/com/starrocks/catalog/Index.java

```diff
@@ -174,12 +236,58 @@ public String toSql() {
 
     public TOlapTableIndex toThrift() {
         TOlapTableIndex tIndex = new TOlapTableIndex();
+        tIndex.setIndex_id(indexId);
         tIndex.setIndex_name(indexName);
         tIndex.setColumns(columns);
         tIndex.setIndex_type(TIndexType.valueOf(indexType.toString()));
         if (columns != null) {
             tIndex.setComment(comment);
         }
+        if (properties != null) {
```

Comment:
I see that there are no changes in the backend, we can introduce thrift and protoco changes in the following PR.